src/table/components/TablePaginationActions.jsx

```diff
@@ -8,48 +8,8 @@ import FirstPageIcon from '@mui/icons-material/FirstPage';
 import KeyboardArrowLeft from '@mui/icons-material/KeyboardArrowLeft';
 import KeyboardArrowRight from '@mui/icons-material/KeyboardArrowRight';
 import LastPageIcon from '@mui/icons-material/LastPage';
-import { makeStyles } from '@mui/styles';
 import { handleLastTab } from '../utils/handle-key-press';
 
-const useStyles = makeStyles({
-  root: {
-    flexShrink: 0,
-    paddingLeft: '24px',
-    transform: 'translate(0px, 4px)',
-  },
-  paginationActionButton: {
-    color: 'rgba(0, 0, 0, 0.54)',
-  },
-  disabled: {
-    color: 'rgba(0, 0, 0, 0.3)',
-    cursor: 'default',
-  },
-  focused: {
-    color: '#404040 !important',
-  },
-  caption: {
-    fontSize: '14px',
-    color: 'inherit',
-    width: 'fit-content',
-    position: 'relative',
-    transform: 'translate(0px, 12px)',
-    paddingRight: '8px',
-    height: '30px',
-  },
-  formControl: {
-    flexDirection: 'row',
-  },
-  dropdown: {
-    cursor: 'pointer',
-    minWidth: '16px',
-    maxHeight: '32px',
-    transform: 'translate(0px, 6px)',
-  },
-  input: {
-    padding: '6px 32px 7px 8px',
-  },
-});
```

Comment:
Not all of these are accounted for now it seems? We don't need all?

Reply:
Some are in sx, like paginationActionButton, https://github.com/qlik-oss/sn-table/pull/435/files#diff-853b31539cf11383077f950bc4b8804579281e297cd09c35704d8af9abfbe809R53. As they are dynamic.
Most of them are in theme, like caption, https://github.com/qlik-oss/sn-table/pull/435/files#diff-e855eaee3db92d1081a0c3f863ab0731db73167e8cbe17399706637105258cf8R25. As they are static.